src/app/modules/admin/component/employees/employees.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="table-background border">
+<div class="table-background border mt-3">
```

Comment:
You do not need to add that, especially in top element of component, 'cuz everywhere is same margin.